src/components/procedure/_customize_pr_mr_comments.mdx

```diff
@@ -0,0 +1,12 @@
+You can customize the comments Semgrep leaves on your {props.comment_type}. Custom comments allow you to direct your teams to the resources they need to handle the vulnerabilities Semgrep identifies in their code.
+
+To provide custom {props.comment_type} comments:
+
+1. Sign in to [<i class="fas fa-external-link fa-xs"></i> Semgrep AppSec Platform](https://semgrep.dev/login?).
+1. Navigate to **Settings > General > Global**.
+1. Go to the **Custom PR/MR comments footers** section.
+1. Provide a custom comment for each Semgrep product whose findings you want to generate a PR comment. Semgrep supports both plaintext and Markdown links in your message.
+1. Click **Save changes**.
+
+![Custom comment configuration in Semgrep AppSec Platform](/img/custom-comments-configuration.png#md-width)
```

Comment:
Ty for adding this in! Do you think it'll also be helpful to attach a picture of what the comment will look like? I know some folks had questions about it, and thought perhaps adding a picture of how the comment is rendered may be helpful?

Reply:
<img width="802" alt="Screenshot 2025-06-13 at 4 56 32 PM" src="https://github.com/user-attachments/assets/058ea6ce-0f64-4e2b-820c-afacd8c1a21a" />

Here's a screenshot! I've outlined the customizable part in red, but if you think it's better without that, happy to send that too :)